doc/radosgw/archive-sync-module.rst

```diff
@@ -4,34 +4,30 @@ Archive Sync Module
 
 .. versionadded:: Nautilus
 
-This sync module leverages the versioning feature of the S3 objects in RGW to
-have an archive zone that captures the different versions of the S3 objects
-as they occur over time in the other zones.
+The Archive Sync module uses the RGW versioning feature of S3 objects to
+maintain an archive zone that captures successive versions of objects
+as they are updated in other zones.  Archive zone objects can
+only be removed through gateways associated with the archive zone.
```

Comment:
I suggest changing the position of "only" so that the sentence reads like this: "...be removed only through gateways associated..."

Reply:
zu Ende